html-css/override-extend-variables/step4-instructions.md

```diff
@@ -1,11 +1,11 @@
 ## Add a PatternFly class for the layout
-
-Now that we have an updated Card design, let's add several Cards and use the gallery layout to organize them on the page. 
```

Comment:
sb: Now that you have an updated Card design, add several Cards and use the gallery layout to organize them on the page.